docs/docs/02-guide/03-hello-world.md

```diff
@@ -0,0 +1,101 @@
+---
+description: Build your first blockchain and your first Cosmos SDK query.
+title: Hello World
+---
+
+# "Hello world!" Blockchain Tutorial with Ignite CLI
+
+**Introduction**
+
+In this tutorial, you'll build a simple blockchain using Ignite CLI that responds to a custom query with "Hello %s!", where %s is a name provided in the query. 
+This will enhance your understanding of creating custom queries in a Cosmos SDK blockchain.
+
+## Setup and Scaffold
+
+1. **Create a New Blockchain:**
+
+```bash
+ignite scaffold chain hello
+```
+
+2. **Navigate to the Blockchain Directory:**
+
+```bash
+cd hello
+```
+
+## Adding a Custom Query
+
+- **Scaffold the Query:**
+
+```bash
+ignite scaffold query say-hello name --response name
+```
+
+This command generates code for a new query, `say-hello`, which accepts a name, an input and returns it in the response.
+
+- **Understanding the Scaffolded Code:**
+
+	- `proto/hello/hello/query.proto`: Defines the request and response structure.
```

Comment:
Did we miss `x/hello/client/cli/query.go`?

Reply:
It's not so relevant here.